src/py_dss_interface/DSS.py

```diff
@@ -3,13 +3,15 @@
 import os
 import json
 import pathlib
+
+from . import Circuit, DSSInterface, Loads, LoadShapes, Solution, Text, Topology, Transformers, XYCurves
 from .utils.System import System
-from py_dss_interface import Circuit, DSSInterface, Loads, LoadShapes, Solution, Text, Topology, Transformers, XYCurves
+# from py_dss_interface import Circuit, DSSInterface, Loads, LoadShapes, Solution, Topology, Transformers, XYCurves
 
 DLL_NAME = "OpenDSSDirect.dll"
 
 
-class DSS:
+class DSS(Circuit, DSSInterface, Loads, LoadShapes, Solution, Text, Topology, Transformers, XYCurves):
```

Comment:
With this approach we can call all methods after instantiate class DSS.
In the last commit we can call like this: 
```
from py_dss_interface import DSS
dss = DSS()
dss_file= r"path_for_your_case"
dss.Text.text("compile {}".format(dss_file))
```
But now, after requests, we need do this:
from py_dss_interface import DSS
```
dss = DSS()
dss_file= r"path_for_your_case"
dss.text("compile {}".format(dss_file))
```